drbd/global_confs.sls

```diff
@@ -39,6 +42,9 @@
         dialog_refresh: {{ drbd.global.dialog_refresh }}
         quorum: "{{ drbd.common.options.quorum }}"
         multi_primaries: "{{ drbd.common.net.multi_primaries }}"
+        max_buffers: "{{ drbd.common.net.max_buffers}}"
```

Comment:
Most probably doesn't affect, but it is strange that here the values are set as string, and above as numbers

Reply:
It is because the input also can be strings like "4k", "2k" here... Equal to "4096" and "2048"